### ISSUE TITLE
fully offline museum display issues

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#680]
Museums can now install the docker images to get an exhibit, but we haven't thought of a solution for the wikipedia links etc. and how they can be made to work in this context.

**Comment by jrosindell** [comment]
"Is there a way to disable the link to the article within the information page? (Or be able to close the article?) Tapping that link seems to be a dead end in kiosk mode with no means to close or go back."

**Comment by hyanwong** [comment]
I think we can disable the wikipedia tab entirely, no?

**Comment by jrosindell** [comment]
Yes we can but I think we need some settings for this in the codebase so that a user of the docker image as an MD can deploy and have it working offline without having to do any tweaking of the code themselves.
